.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release
+on:
+  push:
+    branches: [ master ]
+    tags: ["*"]
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.1.0
+        with:
+          fetch-depth: 100
+      - name: Fetch tags
+        run: git fetch --depth=100 origin +refs/tags/*:refs/tags/*
+      - uses: olafurpg/setup-scala@v7
+        with:
+          java-version: adopt@1.11
+      - uses: olafurpg/setup-gpg@v2
+      - name: sbt ci-release ${{ github.ref }}
+        run: ./sbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
```

Comment:
Where do we set these secrets, please?

Reply:
in the settings of `https://github.com/avast/grpc-json-bridge`
(but that still has to be done, in order for https://github.com/avast/grpc-json-bridge/pull/105 to mergable; I'm waiting on Sonatype to give me the necessary credentials)